tests/xtro-sharpie/ios.pending

```diff
@@ -6,6 +6,12 @@
 ## Deprecated in xcode8 beta1: "Use -[CXAction fulfill] instead"
 !missing-selector! CXCallAction::fulfillWithResponse: not bound
 
+## Deprecated in xcode8 beta3: "Authorization is no longer required"
+!missing-enum! CXAuthorizationStatus not bound
+
+## Better name in enum CXPlayDTMFCallActionType == CXPlayDtmfCallActionType
```

Comment:
there's some (very basic [1]) code in `Helpers.cs` to match enums.

[1] I have been considering adding them to a text file, one entry per file, e.g.`X=x`